java/org/apache/jasper/Options.java

```diff
@@ -47,6 +47,13 @@
      */
     public boolean getKeepGenerated();
 
+    /**
+     * Returns the Value of JSPWhiteSpaceTrimming Flag
```

Comment:
Provide a description that explains the impact of enabling this flag.  It's somewhat clear from the name, but being explicit helps.

Reply:
Yes, I think I can add some description there

Reply:
Resolved with latest push